### PR TITLE
Adding issue templates to reflect #356

### PR DESCRIPTION
**Pull request by boronhub**
Issue #356 addressed the need for better issue tracking in this repository, and these issue templates address :

- Bugs/Performance issues
- Build/Installation issues
- Documentation Requests/Issues
- Feature Requests

There is also a general issue template for those not in one of these categories. I have not altered the default issue template, as I expect the repository maintainers to update it when the move away from JIRA is successful.

### CODE REVIEW

**Comment by boronhub** [comment]
@dan-zheng I hope this PR is of substance

File: .github/ISSUE_TEMPLATE/10-bug-performance-issue.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+name: Bug/Performance Issue
+about: Use this template for reporting a bug or a performance issue.
+
+---
+
+<em>Please make sure that this is a bug. As per our [GitHub Policy](https://github.com/boronhub/swift/blob/master/KNOWN_ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub.
```
Comment (dan-zheng) [inline .github/ISSUE_TEMPLATE/10-bug-performance-issue.md]: Please replace all occurrences of `boronhub/swift` to `tensorflow/swift`:

```suggestion
<em>Please make sure that this is a bug. As per our [GitHub Policy](https://github.com/tensorflow/swift/blob/master/KNOWN_ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub.
```

Same in other files.

**Comment by boronhub** [comment]
@dan-zheng ping

File: .github/ISSUE_TEMPLATE/10-bug-performance-issue.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+about: Use this template for reporting a bug or a performance issue.
+---
+
+<em>Please make sure that this is a bug. As per our [GitHub Policy](https://github.com/tensorflow/swift/blob/master/KNOWN_ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub.</em>
+
+**System information**
+
+- Have I written custom code (as opposed to using a stock example script provided in Swift for TensorFlow):
+- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
+- Swift for Tensorflow installed from (source or binary):
```
Comment (dan-zheng) [inline .github/ISSUE_TEMPLATE/10-bug-performance-issue.md]: Please spell "TensorFlow" with a capital "F":
```suggestion
- Swift for TensorFlow installed from (source or binary):
```
Same everywhere.

**Comment by boronhub** [comment]
@ematejska @dan-zheng I have made the changes.

**Comment by ematejska** [comment]
Yes, I will review in the next day or so.  Sorry for the delay and thanks for your patience :)

**Comment by boronhub** [comment]
@ematejska ping

**Comment by boronhub** [comment]
@ematejska ping

**Comment by boronhub** [comment]
@dan-zheng ping

**Comment by ematejska** [comment]
@boronhub We looked through the bugs that are filed and it looks like in general we don't need so much detailed information to make good progress on the bugs.  So I want to simplify these templates.  Also, we're thinking about how/when to point folks to JIRA.  It might still take a little time to figure out the bug policy.  Please bear with us.

**Comment by ematejska** [comment]
We are going to restructure a bit how bug tracking is done.  I'm closing this because we don't need to update the templates until that is done.

**Pull request closed without merging**